src/main/java/Task.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Represents a Task. Contains a Task constructor, two methods to mark and unmark tasks, toString() method as well as a isMark() method to check if Task is marked
+ */
+public class Task {
+    private boolean mark;
+    public String name;
+
+    /**
+     * Constructor
+     * @param name name of the task
+     */
+    public Task (String name) {
+        this.name = name;
+        this.mark = false;
+    }
+
+    /**
+     * markTask as done
+     */
+    public void markTask () {
+        String markedMessage = "Nice! I've marked this task as done:\n";
+        this.mark = true;
+        System.out.println(markedMessage + "  " + this);
+    }
+
+    /**
+     * unmarkTask
+     */
+    public void unmarkTask() {
```

Comment:
Should this be named in this manner? Perhaps the following would comply with the coding standards better.
```suggestion
    public void setUnmarkedTask () {
```